app/javascript/components/pages/task-list/container.spec.js

```diff
@@ -0,0 +1,509 @@
+import { fromJS, OrderedMap } from "immutable";
+import MUIDataTable, { TableBodyRow } from "mui-datatables";
+
+import { mountedComponent, stub, screen } from "../../../test-utils";
+import { DashboardChip } from "../../dashboard";
+import { ListHeaderRecord } from "../../user/records";
+import { FieldRecord, FormSectionRecord } from "../../record-form/records";
+
+import TaskList from "./container";
+
+describe("<TaskList />", () => {
+  let stubI18n = null;
+  let component;
+stubI18n = stub(window.I18n, "t").withArgs("date.formats.default").returns("%d-%b-%Y");
+
+  it("should render tasks table", () => {
+    mountedComponent(
+        <TaskList />,
+        fromJS({
```

Comment:
Create a constant at the top to reuse this data in the tests.

Reply:
Done